docs/README.md

```diff
@@ -50,6 +44,18 @@ Summarizing as below:
 
 <figure><img src=".gitbook/assets/report_review_process.jpg" alt=""><figcaption></figcaption></figure>
 
+### Quality Assessment Collection Reporting Process (only for SBI)
+
+* The Quality Assessment Collection reporting procedure is very similar to the Compliance Collection process.
+* Using this collection,  partner can collect biometric scores for various groups of individuals (age wise, gender wise, occupation wise etc.).
+* The biometric scores will be given using BQAT SDK.
+* The scores will be used to perform quality assessment of the SBI partner.
```

Comment:
The scores will serve as the basis for evaluating the quality of the SBI partner.